.maintain/build-image.sh

```diff
@@ -22,3 +22,19 @@ docker build \
     --build-arg NODE_TYPE=standalone-node \
     -t local/standalone-node:$target_tag \
     .
+
+# build DIP provider and consumer templates
+PROVIDER_BIN_NAME="dip-provider-node-template"
+docker build \
+	--cache-from $AWS_REGISTRY/kilt-parachain/collator:builder \
+	--cache-from $AWS_REGISTRY/dip-provider-node-template:$target_tag \
```

Comment:
Is this also correct? I have to take cache from the latest image of the same type pushed, and not from the base `kilt-parachain` image, right? The naming is a bit misleading, but I hope I got it right.